lib/commentTreeUtils.es6.js

```diff
@@ -6,7 +6,7 @@ import { COMMENT_LOAD_MORE } from '../models2/thingTypes';
 export function treeifyComments(comments=[]) {
   const commentDict = {};
   comments.forEach(c => {
-    commentDict[c.uuid] = c;
```

Comment:
At this point it's just the json and it's missing the uuid. It's a weird bug in that I can't imagine how it got into this state unless the field was there at some point.